### ISSUE TITLE
Cud 1.105, plugin don't add hotkeys for macros menuitems

### ISSUE THREAD

**Issue by Alexey-T** [kvichans/cuda_macros#19]
@kvichans @tmsg-gh
maybe it worked before 1.105? but now not. plugin runs in "on_start" but it doesn't add hotkeys for my macros. I have 1 macro with Alt+W and menuitem for this macro has no Alt+W. (after Cud restart)

- keys.json has Alt+W
- macros -

```
{
    "ver": "20151204",
    "list": [
        {
            "id": 34882,
            "nm": "tst-edit",
            "rec": "150,m\n150,a\n150,c\n150,r\n150,o\n150,.\n",
            "evl": [
                "ed.cmd(cmds.cCommand_TextInsert,'macro.')"
            ]
        }
    ],
    "dlg_prs": {
        "h_acts": 450,
        "times": 1000,
        "h_list": 450,
        "w_acts": 500,
        "w_btn": 200,
        "w_list": 200
    },
    "tm_ctrl": {
        "rp_ctrl": 1000,
        "tm_wait": 10
    }
}
```

**Comment by Alexey-T** [comment]
keys.json
```
    ]
  },
  "cuda_macros,run,34882" : {
    "name" : "plugin: Macros: tst-edit",
    "s1" : [
      "Alt+W"
    ]
  }
}```

**Comment by Alexey-T** [comment]
Ok, after a fix in Cud 1.105.1 beta, this is not the issue
